sampleSpecs/noerrors.smithy

```diff
@@ -0,0 +1,31 @@
+$version: "2.0"
+
+namespace smithy4s.example
+
+@smithy4s.api#simpleRestJson
+service EchoService {
```

Comment:
I wanted to add a new operation to `pizza.smithy` and make a pizza test for this, but unfortunately PizzaSpec is pretty much hard-wired to a single service, with a list of errors in the service itself - so I thought it would be impossible to reproduce the erroneous behavior without changing how PizzaSpecs are implemented.

One thing came to mind, though: smithy4s (the runtime) doesn't care whether an error comes from the service or the operation, so maybe we could inline these service-wide errors into the operations that need them? (we most likely cover the support of service errors in many other tests).

That way we could merge this into Pizza and have some guarantees that users of PizzaSpec will also implement the right behavior.